src/bot.py

```diff
@@ -2,8 +2,9 @@
 # -*- coding: utf-8 -*-
 
 '''
-@author: AlwaysLivid
+@author: Panagiotis Vasilopoulos ( AlwaysLivid )

```

Comment:
Would you mind removing the spaces between the parentheses and "AlwaysLivid", rebasing the commit and force-pushing?

I'm hoping that you'll also learn something in the process.

Thank you!

Reply:
Done, I don't do automatic commits! 
Let me know if it's ok now.